### ISSUE TITLE
Appimage from version 1.4.4 does not run in Arch Linux

### ISSUE THREAD

**Issue by Gilraiser** [synfig/synfig#2963]
**Bug Description**
When executing the appimage in terminal, the following information is printed:

` /tmp/.mount_knO61x/usr/bin/synfigstudio: /tmp/.mount_knO61x/usr/lib/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /usr/lib/libjack.so.0)`

**To Reproduce**
1. Go to the folder where the appimage is located.
2. Open a terminal or console there.
3. Type in the console ./SynfigStudio-1.4.4.Appimage to run.
4. The message described above will appear.

**Expected behavior**
Synfig studio should be displayed correctly.

**System information:**
   Operating System: Arch Linux                      
   Kernel: Linux 5.15.85-1-lts
   Architecture: x86-64

 - Package: AppImage
 - Synfig version: 1.4.4

**Additional comments**
This is the most similar information I found about the same console message:
https://bbs.archlinux.org/viewtopic.php?id=171351
https://bbs.archlinux.org/viewtopic.php?id=274705
https://stackoverflow.com/questions/44773296/libstdc-so-6-version-glibcxx-3-4-20-not-found
https://github.com/rapidsai/cuml/issues/3383
I have reviewed my settings and I don't think this is the case. My version of gcc is 12.2.0-1 of the group base-devel from the core Arch Linux official repository.

Thank you very much for improving Synfig day by day. I hope the information is useful for everyone :)

**Comment by ice0** [comment]
@morevnaproject 

You need to include `libjack` or remove `libstdc++.so`.

**Comment by BobSynfig** [comment]
Please try this patch: https://gist.github.com/BobSynfig/b861ead8ea57df22d601edb22324d6a6
Just replace AI_SRC and AI_DST with proper Synfig AppImage names ;)

**Comment by ice0** [comment]
@BobSynfig 
`--appimage-extract` will not work, because we still on AppImage v1.

**Comment by BobSynfig** [comment]
@ice0
Yeah, right, I updated the patch for 1.4.0 to 1.4.4, now it works
https://github.com/BobSynfig/PatchSynfig140/blob/main/PatchSynfig144.sh
Don't forget to check the "options" in the beginning of the script

**Comment by Gilraiser** [comment]
Thank you very much friends. The patch worked

**Comment by morevnaproject** [comment]
> @ice0 Yeah, right, I updated the patch for 1.4.0 to 1.4.4, now it works https://github.com/BobSynfig/PatchSynfig140/blob/main/PatchSynfig144.sh Don't forget to check the "options" in the beginning of the script

Can we include parts of this patch to morevna-builds? - https://github.com/morevnaproject/morevna-builds/blob/master/env-builder-data/build/script/packet/synfigstudio-master-appimage.sh

**Comment by BobSynfig** [comment]
With pleasure, it would permit also to convert Appimage v1 to v2 with compression (we could avoid to do a pass as v1.
Note that there are some elements inside to change mistaken symlinks (could be done with the other scripts, it would be more clean.
It also removes unnecessary translations (no desirable directly).
